README.md

```diff
@@ -123,50 +124,50 @@ for help about a specific subcommand.
 
 1. Download build dependencies:
    - **With Nix**: If you have [Nix](https://nixos.org/nix) installed:
-     ```
+     ```shell
      $ nix-shell shell.nix
      ```
      to be dropped in a shell, ready to build.
    - **Without Nix**: otherwise, follow [this guide][rust-guide] to install Rust
      and Cargo first.
 2. Build Nickel:
-   ```
+   ```shell
    $ cargo build
    ```
    And voilà! Generated files are placed in `target/debug`.
 
 
 1. *(optional)* make a symbolic link to the executable:
-  ```
+  ```shell
   $ ln -S nickel target/debug/nickel
   ```
 
 
 ### Tests
 
-```
+```shell
 $ cargo test
 ```
 
 ### Documentation
 
 1. Build the doc:
-  ```
+  ```shell
   $ cargo doc --no-deps
   ```
 2. Open the file `target/doc/nickel/index.html` in your browser.
 
 ### Examples
 
 You can find examples in
-[`examples`](./examples). Note
+[`/examples`](./examples) directory. Note
```

Comment:
I think it was  better before, but even if it is to be changed I think we need the dot here:
```suggestion
[`./examples`](./examples) directory. Note
```

Reply:
I find it weird-looking to be both an inline code (backticks) AND a link:
![image](https://user-images.githubusercontent.com/1204125/128839923-793068b4-660c-4a4b-8cdc-34a028d4bf40.png)

I'd go with:
* either don't make it inline code, i.e. only write [examples](./examples)
* or make it explicit it refers to the directory (in which case I agree with your suggestion of adding the dot), i.e. [`./examples`](./examples)

Reply:
Yeah, I see. It's fine either way for me; let's see what @yannham prefers :slightly_smiling_face: